201-vmss-windows-webapp-dsc-autoscale/azuredeploy.parameters.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmSku": {
+      "value": "Standard_A1"
+    },
+    "windowsOSVersion": {
+      "value": "2012-R2-Datacenter"
+    },
+    "vmssName": {
+      "value": "GEN-UNIQUE-5"
+    },
+    "instanceCount": {
+      "value": 3
+    },
+    "adminUsername": {
+      "value": "vmssadmin"
+    },
+    "adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "_artifactsLocation": {
+      "value": null
```

Comment:
I added the parameters back in to match the defaults in the template. I left _artifactsLocationSasToken in with a value of null since it was the only one remaining as null and that way if someone pulls it down locally they have all the parameters included. Let me know if that's not ok and I can certainly remove it.